src/index.coffee

```diff
@@ -13,4 +13,5 @@ module.exports = (app, options) ->
     autoMigrate = require './' + autoMigrate
     autoMigrate(app, options)
   else
-    Promise.reject(new TypeError 'component not enabled')
+    Promise.resolve().then (res) ->
```

Comment:
Got it, by the way the component is very useful. I was thinking that instead of returning an error, you could return to the console and indication that the component is off. Therefore my suggestion of the resolve(). Any thoughts?